### PR TITLE
Fix goheader linter

### PR DESCRIPTION
**Pull request by tompsota**
Resolved warnings from goheader linter

**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/header.tpl [hunk 1]
```diff
@@ -1,6 +1,5 @@
 Copyright The Kubernetes NMState Authors.
```
Comment (phoracek) [inline hack/header.tpl:2]: You have removed a line from the template, but in all the rendered headers there are two empty lines. Is it on purpose?

**Review by phoracek — COMMENTED** [review]
/hold

It does not look consistent to me. Feel free to cancel the hold once answered

**Comment by phoracek** [comment]
@tompsota the merge was faster, but would you please see my comment above?

File: hack/header.tpl (hunk 1 above)
Comment (tompsota) [reply hack/header.tpl:2]: Yes, it's on purpose. For some reason the goheader linter fails if there are 2 newlines in the template (most likely a bug). I tried to point this out in the commit message of 6c5d6b91a2d830085c87fde1f9f7e1fc78b407e2 , but maybe I was too brief.

**Pull request merged**